pkg/controllers/management/clusterconnected/clusterconnected.go

```diff
@@ -103,6 +103,12 @@ func (c *checker) checkCluster(cluster *v3.Cluster) error {
 		return nil
 	}
 
+	// wait to update the connected condition until it is bootstrapped
+	if !v3.ClusterConditionPreBootstrapped.IsTrue(cluster) {
```

Comment:
so, this is going to potentially affect imported and rke1 clusters... need to check the v3 cluster to see if it is a provisioningv2/CAPR cluster before adding this check here.